### DISCUSSION TITLE
Running Get-WinGetPackage in Service gives error: cmdlet not supported in Windows Powershell even tough when i run it perfectly through terminal

### DISCUSSION THREAD

**Discussion by jonygomes06** [microsoft/winget-cli#4832] [Q&A]
I have an exe that is intended to run as a service but i can also run it through terminal.

When i run that exe through terminal, just like a cmd command, i can perfectly use Get-WinGetPackage, the same works when i use the Get-WinGetPackage directly in terminal.

But when i use my exe as a service i get this error: Get-WinGetPackage : This cmdlet is not supported in Windows PowerShell.

Why this happens?

Im using C:\Windows\system32\WindowsPowerShell\v1.0\powershell.exe in all scenarios

Thank you for your time

**Comment by denelon** [comment]
Related to:
* https://github.com/microsoft/winget-cli/issues/4838
